src/App.js

```diff
@@ -1,25 +1,101 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react'
+import styled from 'styled-components'
```

Comment:
My personal preference:
```suggestion
import styled from 'styled-components/macro'
```

Reply:
mine too, i just forgot...